test/e2e/set/scyllacluster/scyllacluster_ingress.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	scyllafixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scylla"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("ScyllaCluster Ingress", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should create ingresses", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc.Spec.Datacenter.Racks[0].Members = 2
+		sc.Spec.Domains = []string{"private.nodes.scylladb.com", "public.nodes.scylladb.com"}
+		sc.Spec.ExposeOptions = &scyllav1.ExposeOptions{
+			Ingress: &scyllav1.IngressOptions{
+				IngressClassName: "my-ingress-controller",
+				Annotations: map[string]string{
+					"my-annotation": "my-annotation",
```

Comment:
tiny nit: best to use distinct string to test we don't switch key and value accidentally